Migrate docs to the new kyma cli (#1542)
kyma-project/serverless: docs/user/tutorials/01-90-set-asynchronous-connection.md

## Prerequisites

- [Kyma CLI](https://github.com/kyma-project/cli)

> - [Eventing, Istio and API-Gateway components installed](https://kyma-project.io/#/02-get-started/01-quick-install)

Reviewer: ```suggestion
- [Eventing, Istio, and API-Gateway modules added](https://kyma-project.io/#/02-get-started/01-quick-install)
```

   The event object provides convenience functions to build and publish events. To send the event, build the Cloud Event. To learn more, read [Function's specification](../technical-reference/07-70-function-specification.md#event-object-sdk). In addition, your **eventOut.source** key must point to `“kyma”` to use Kyma in-cluster Eventing.
   There is a `require('axios')` line even though the Function code is not using it directly. This is needed for the auto-instrumentation to properly handle the outgoing requests sent using the `publishCloudEvent` method (which uses `axios` library under the hood). Without the `axios` import the Function still works, but the published events are not reflected in the trace backend.

3. Apply your emitter Function:

   ```bash
   kyma alpha function create emitter --source handler.js --dependencies package.json
   ```

   Your Function is now built and deployed in Kyma runtime. Kyma exposes it through the APIRule. The incoming payloads are processed by your emitter Function. It then sends the sanitized content to the workload that subscribes to the selected event type. In our case, it's the receiver Function.

> 4. Export the cluster domain address (read how to [set up your cluster domain](https://kyma-project.io/#/api-gateway/user/tutorials/01-10-setup-custom-domain-for-workload)):

Reviewer: ```suggestion
4. Export the cluster domain address.
```

> 4. Export the cluster domain address (read how to [set up your cluster domain](https://kyma-project.io/#/api-gateway/user/tutorials/01-10-setup-custom-domain-for-workload)):

Reviewer: Can you move the tutorial link to the Prerequisites section?

   ```bash
   export KYMA_DOMAIN={KYMA_DOMAIN_VARIABLE}
   ```

> 5. Expose Function by creating the APIRule custom resource:

Reviewer: ```suggestion
5. Expose Function by creating the APIRule CR:
```
